src/test.rs

```diff
@@ -116,13 +116,13 @@
 //! stack trace to the line in the test code that panicked: entry 9 in the
 //! stack trace pointing to line 652 of the file `example.rs`.
 
-#[cfg(feature = "use_heap")]
-use crate::bits;
+use crate::{bits, digest, error};
 
-use crate::{digest, error};
+#[cfg(feature = "alloc")]
```

Comment:
If `#[cfg(feature = "alloc")]` is added to `lib.rs` then these annotations are not needed in this file (here and elsewhere).